Stellar-operation-create-manage-limits-request.x

```diff
@@ -15,56 +15,68 @@ namespace stellar
     Result: CreateManageLimitsRequestResult
 */
 
+//: CreateManageLimitsOp is used to create reviewable request which on approval will update limits set in the system
 struct CreateManageLimitsRequestOp
 {
+    //: Body of the UpdateLimits reviewable request to be created
     LimitsUpdateRequest manageLimitsRequest;
 
-	uint32* allTasks;
+    //: (optional) Bit mask whose flags must be cleared in order for ManageLimits request to be approved, which will be used
+    //: instead of key-value by key `limits_update_tasks`
+    uint32* allTasks;
+    //: ID of the LimitsUpdateRequest
+    //: If `requestID == 0` - operation creates new limits entry, otherwise - updates existing
     uint64 requestID;
 
-
-	// reserved for future use
-	union switch (LedgerVersion v)
-	{
-	case EMPTY_VERSION:
-		void;
-	}
-	ext;
-
+    //: reserved for future use
+    union switch (LedgerVersion v)
+    {
+    case EMPTY_VERSION:
+        void;
+    } ext;
 };
 
 /******* CreateManageLimits Result ********/
 
+//: Result codes for CreateManageLimitsRequest operation
 enum CreateManageLimitsRequestResultCode
 {
     // codes considered as "success" for the operation
+    //: ManageLimitsRequest was successfully created and operation was successfully applied
     SUCCESS = 0,
 
     // codes considered as "failure" for the operation
-	MANAGE_LIMITS_REQUEST_REFERENCE_DUPLICATION = -1,
+    //: There is another manage limits request for source account
+    MANAGE_LIMITS_REQUEST_REFERENCE_DUPLICATION = -1,
+    //: There is no request with such ID
     MANAGE_LIMITS_REQUEST_NOT_FOUND = -2,
-    INVALID_CREATOR_DETAILS = -3, // details must be valid json
-	LIMITS_UPDATE_TASKS_NOT_FOUND = -5,
-    NOT_ALLOWED_TO_SET_TASKS_ON_UPDATE = -6, // can't set allTasks on request update
+    //: Details must be valid json
+    INVALID_CREATOR_DETAILS = -3,
+    //: Cannot load tasks fot the CreateManageLimitsRequest or configuration restricts to create manage limits request
+    LIMITS_UPDATE_TASKS_NOT_FOUND = -5,
+    //: Cannot set allTasks on rejected request update
+    NOT_ALLOWED_TO_SET_TASKS_ON_UPDATE = -6,
+    //: 0 not allowed for allTasks if set, or not allowed for value entry, got by key limits_update_tasks:<asset_code>
```

Comment:
0 value is either not allowed for `allTasks` or for the value entry received by key limits_update_tasks:<asset_code>